hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsRestOperation.java

```diff
@@ -157,7 +175,7 @@ void execute() throws AzureBlobFileSystemException {
       requestHeaders.add(httpHeader);
     }
 
-    int retryCount = 0;
+    retryCount = 0;
```

Comment:
this. prefix?

Reply:
this. usage has been advised against many times in previous PRs by Steve to align with diff IDE. Retaining current code.